services/drupal/web/modules/custom/epa_wysiwyg/epa_wysiwyg.module

```diff
@@ -93,3 +94,68 @@ function template_preprocess_epa_file_link(&$variables) {
   $variables['url'] = Url::fromUri($url, $options);
   $variables['link'] = Link::fromTextAndUrl($variables['link_text'], $variables['url'])->toRenderable();
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function epa_wysiwyg_form_editor_link_dialog_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Alter only the form with ID 'editor_link_dialog'.
+  if ($form_id !== 'editor_link_dialog') {
```

Comment:
Looking at this again, this seems redundant since we're using the `FORM_ID` hook.